### ISSUE TITLE
[CI] Fix failing iOS end-to-end tests

### ISSUE THREAD

**Issue by hramos** [facebook/react-native#19538]
Tests are currently failing with:

```
error: bundling failed: TypeError: Cannot read property 'bindings' of null
    at Scope.moveBindingTo (/private/tmp/react-native-fsA0i4iC/EndToEndTest/node_modules/@babel/traverse/lib/scope/index.js:978:13)
    at BlockScoping.updateScopeInfo (/private/tmp/react-native-fsA0i4iC/EndToEndTest/node_modules/babel-plugin-transform-es2015-block-scoping/lib/index.js:364:17)
    at BlockScoping.run (/private/tmp/react-native-fsA0i4iC/EndToEndTest/node_modules/babel-plugin-transform-es2015-block-scoping/lib/index.js:330:12)
    at PluginPass.BlockStatementSwitchStatementProgram (/private/tmp/react-native-fsA0i4iC/EndToEndTest/node_modules/babel-plugin-transform-es2015-block-scoping/lib/index.js:70:24)
    at newFn (/private/tmp/react-native-fsA0i4iC/EndToEndTest/node_modules/@babel/traverse/lib/visitors.js:237:21)
    at NodePath._call (/private/tmp/react-native-fsA0i4iC/EndToEndTest/node_modules/@babel/traverse/lib/path/context.js:65:20)
    at NodePath.call (/private/tmp/react-native-fsA0i4iC/EndToEndTest/node_modules/@babel/traverse/lib/path/context.js:40:17)
    at NodePath.visit (/private/tmp/react-native-fsA0i4iC/EndToEndTest/node_modules/@babel/traverse/lib/path/context.js:100:12)
    at TraversalContext.visitQueue (/private/tmp/react-native-fsA0i4iC/EndToEndTest/node_modules/@babel/traverse/lib/context.js:142:16)
 BUNDLE  [ios, dev] ./index.js ░░░░░░░░░░░░░░░░ 0.0% (0/1), failed.
```

Example: https://circleci.com/gh/facebook/react-native/44078

**Comment by AndrewJack** [comment]
related? https://github.com/babel/babel/issues/7627

**Comment by hramos** [comment]
This is because we need to use the react-native-babel-preset v5.0.0 which is tagged as @next instead of @latest on npm in order to avoid breaking `react-native init` for people not on master (see https://github.com/facebook/react-native/commit/34bd776af2f2529009188fced91083203a48fd40).

I think we can temporarily have `init.js` use v5.0.0 in order to fix master, and once that hits a stable release, we can flip @latest on npm to v5.0.x.

**Comment by hramos** [comment]
Waiting for Circle CI to finish running, but so far it looks like the error is resolved when we explicitly use v5.0.0: https://circleci.com/gh/hramos/react-native/2642
